src/ontology/mondo-edit.obo

```diff
@@ -574661,6 +574663,15 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7379" xsd:anyURI
 
+[Term]
+id: MONDO:1040004
+name: PIK3R2-related overgrowth spectrum
+def: "Abnormal brain overgrowth resulting from pathogenic gain-of-function variants in the PIK3R2 gene. The variants can be germline or somatic." [https://clinicalgenome.org/affiliation/50020/, https://orcid.org/0000-0002-0587-4693, PMID:22729224, PMID:26860062, PMID:27854409, PMID:32856318]
```

Comment:
This definition suggests that there is a disease called "Abnormal brain overgrowth". Do we have such disease? 
Could we use the genus/parent term "overgrowth syndrome" with the differentia "PIK3R2 variants"?

Reply:
Comment addressed.